pkg/virt-handler/vm_test.go

```diff
@@ -35,8 +35,9 @@ import (
 	k8sruntime "k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/testing"
 
-	api2 "kubevirt.io/client-go/api"
+	pkg_controller "kubevirt.io/kubevirt/pkg/controller"
```

Comment:
Please use something more consistent. E.g `k6scontroller`

Reply:
Thanks, changed import name to `virtcontroller` (used widely in other files).